docs/dataset-overviews/anthropogenic-slcf-co2-emissions.md

```diff
@@ -1,182 +1,191 @@
+
+Gridded aircraft emissions are also supplied in a separate file with 25 vertical layers (using the CMIP5 historical emissions Lamarque et al. 2010, as drawn from Lee et al., as a template). Note that aircraft emissions are zero in early years, but files are provided for all years consistency.
+
+The gridded emissions incorporate a monthly seasonal cycle by sector drawing largely from the [ECLIPSE project](https://iiasa.ac.at/models-tools-data/global-emission-fields-of-air-pollutants-and-ghgs), [Carbon Tracker](https://carbontracker.org/), and [EDGAR](https://edgar.jrc.ec.europa.eu/).
+
+VOC speciation is provided at the 23 species resolution from [EDGAR](https://edgar.jrc.ec.europa.eu/).
+
+### Variables provided
+
+Subsequent CEDS data releases since August 2016 are in a format of one variable per data file, with the sectors included as a dimension of the variable's data.
+
+The sectors in the netCDF files (other than aviation) are:
+
+| Sector | Description                                                             |
+|--------|-------------------------------------------------------------------------|
+| 0: AGR | Non-combustion agricultural sector                                      |
+| 1: ENE | Energy transformation and extraction                                    |
+| 2: IND | Industrial combustion and processes                                     |
+| 3: TRA | Surface Transportation (Road, Rail, Other)                              |
+| 4: RCO | Residential, commercial, and other                                      |
+| 5: SLV | Solvents                                                                |
+| 6: WST | Waste disposal and handling                                             |
+| 7: SHP | International shipping (including VOCs from oil tanker loading/leakage) |
+
+Within the netCDF files the sector ids are:
+
+`sector:ids = "0: Agriculture; 1: Energy; 2: Industrial; 3: Transportation; 4: Residential, Commercial, Other; 5: Solvents production and application; 6: Waste; 7: International Shipping" ;`
+
+### Supplementary Data
+
+For use in setting aerosol size distribution and additional speciation (if desired), an auxiliary dataset providing emissions from solid biomass combustion is also provided. Note that these are a subset of emissions in the main files. These data, therefore, should NOT be added to the emissions in the main files. (Note that no data files for CO2 emissions from solid biomass are released, as CO2 emissions from CEDS are from fossil fuels only.)
+
+Supplementary checking `.csv` text files that provide total global mass for each sector, month, and year are also available as well as global seasonal diagnostic plots can be found [here](https://zenodo.org/records/14145000).
```

Comment:
As a note, checksum generally refers to the checksum of the file so I would use different language to avoid confusing people (although I get that the idea is similar).